modules/rostests/apitests/shell32/ShellExecuteW.cpp

```diff
@@ -44,7 +44,7 @@ TestShellExecuteW()
         LPCWSTR lpWinTitle = L"Display Properties";
         Sleep(1000);
         hWnd = FindWindowW(NULL, lpWinTitle);
-        PostMessage(hWnd, WM_QUIT, 0, 0);
+        PostMessage(hWnd, WM_SYSCOMMAND, 0xF060, 0);  // Terminate Window
```

Comment:
But then why not using the symbol "SC_CLOSE" instead?

Reply:
I did not think about doing it previously, but was only concerned with its functionality. Now I have made the changes to do it using SC_CLOSE. Thanks.